05-visualisation-ggplot2.Rmd

```diff
@@ -152,7 +152,7 @@ ggplot(surveys_complete)
 - define aestetics (`aes`), that maps variables in the data to axes on the plot
      or to plotting size, shape color, etc.,
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
So why does the command not work? Shouldn't we fix the command so that it will execute without error, instead of preventing it from getting executed but displaying the command in the rendered lesson anyway?

BTW you can also make R proceed despite error.